dali/python/setup.py.in

```diff
@@ -14,7 +14,7 @@
 
 from setuptools import setup, find_packages
 
-setup(name='nvidia-dali',
+setup(name='nvidia-dali@CU_DALI_VERSION@',
```

Comment:
question, do we want to have the cuda version in the name of the package or just in the name of the wheel?

Reply:
In the name of the wheel is enough, I don't think we want people to have them both side by side.

Reply:
here the package name will be different for different CUDA versions. If we just want to rename the wheel file we can do it in the `Dockerfile`.
Also, we should test if this causes any trouble with `nvidia-dali-tf-plugin` finding `nvidia-dali`

Reply:
I added version to plugin name and requirements as well.